src/components/themed/FioRequestRow.tsx

```diff
@@ -201,15 +201,13 @@ export const FioRequestRow = connect<StateProps, {}, OwnProps>(
     }
     const tokenCode = fioRequest.content.token_code.toUpperCase()
     try {
-      const { allCurrencyInfos } = state.ui.settings.plugins
-      const plugin = allCurrencyInfos.find(plugin => {
-        const { currencyCode: pluginCurrencyCode } = plugin
-        if (pluginCurrencyCode == null) return false
-        return pluginCurrencyCode.toUpperCase() === fioRequest.content.chain_code.toUpperCase()
-      })
-
-      if (plugin == null) throw new Error(`No plugin match for this chain code - ${fioRequest.content.chain_code.toUpperCase()}`)
-      displayDenomination = getDisplayDenomination(state, plugin.pluginId, tokenCode)
+      const chainCode = fioRequest.content.chain_code.toUpperCase()
```

Comment:
Rebase after https://github.com/EdgeApp/edge-react-gui/pull/3778 is merged since that PR fixes a bug and removes allCurrencyInfos